javascript/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@dmsnell/diff-match-patch",
+  "version": "1.0.0",
+  "description": "Efficient and human-readable text diffs with helpers",
+  "keywords": [
+    "diff",
+    "diff-match-patch",
+    "google-diff-match-patch"
+  ],
+  "exports": {
+    "import": "./diff_match_patch_uncompressed.js",
+    "require": "./index.mjs"
```

If we were to go with both forms, it looks like this is flipped - [import should be esm and require should be cjs](https://nodejs.org/api/packages.html#conditional-exports):

```suggestion
    "import": "./index.mjs",
    "require": "./diff_match_patch_uncompressed.js"
```

---

Whoops! I wonder why my tooling didn't pick this up. I guess bundlers are just used to things being weird and make it work somehow.